mythril/start_time.py

```diff
@@ -0,0 +1,7 @@
+from time import time
+from mythril.support.support_utils import Singleton
+
+
+class StartTime(metaclass=Singleton):
```

Comment:
documentation 😇

Reply:
Also I think we should move this to support

Reply:
Moved